node/config/config.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+type Config struct {
+	P2P P2P
```

Comment:
Last comment - should we store the copy or a pointer to the conf? @Wondertan

Reply:
pointer might be better if other packages in the node might need to update the conf based on env?

Reply:
I agree.